src/schema/rules/checks/events.yaml

```diff
@@ -27,3 +27,15 @@ StimulusFileMissing:
     - columns.stim_file != null
   checks:
     - exists(columns.stim_file, "stimuli") == length(columns.stim_file) - count(columns.stim_file, "n/a")
+
+SortedOnsets:
+  issue:
+    code: EVENT_ONSET_ORDER
+    message: |
+      The onset column in events.tsv files should be sorted.
+    level: warning
+  selectors:
+    - suffix == "events"
+    - extension == ".tsv"
+  checks:
+    - sorted(columns.onset) == columns.onset
```

Comment:
@rwblair Do you agree?

Reply:
n/a would be the only thing to mess it up if allowed. Otherwise looks good.

Reply:
Good point. I think it being a warning should make that an annoyance, not a show-stopper. Will add a comment, though.